### PR TITLE
add com.gryphel.mini_vmac

### PR DESCRIPTION
**Pull request by rs0vere**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by rs0vere** [comment]
bot, build com.gryphel.mini_vmac

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: why x86_64 only?

File: com.gryphel.mini_vmac.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        path: com.gryphel.mini_vmac.desktop
+    buildsystem: simple
+    build-commands:
+      - gcc setup/tool.c -o setup_t
+      - ./setup_t -t lx64 > ./setup.sh
+      - . ./setup.sh; make
+      - mkdir -p $FLATPAK_DEST/bin/
+      - install -Dm755 minivmac $FLATPAK_DEST/bin/
+      - install -Dm644 com.gryphel.mini_vmac.metainfo.xml -t $FLATPAK_DEST/share/metainfo
+      - install -Dm644 com.gryphel.mini_vmac.png -t $FLATPAK_DEST/share/icons/hicolor/32x32/apps
```
Comment (hfiguiere) [inline com.gryphel.mini_vmac.yml:36]: minimum icon size required is 128x128.

File: com.gryphel.mini_vmac.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  </categories>
+  <keywords>
+    <keyword>emulator</keyword>
+  </keywords>
+
+  <releases>
+    <release version="36.04" date="2020-12-13"/>
+  </releases>
+
+  <content_rating type="oars-1.0"/>
```
Comment (hfiguiere) [inline com.gryphel.mini_vmac.metainfo.xml:30]:
```suggestion
  <content_rating type="oars-1.1"/>
```

File: com.gryphel.mini_vmac.metainfo.xml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      <image>https://www.gryphel.com/d/minivmac/screens/Snow.gif</image>
+    </screenshot>
+    <screenshot>
+      <image>https://www.gryphel.com/d/minivmac/screens/Win7.gif</image>
+    </screenshot>
+    <screenshot>
+      <image>https://www.gryphel.com/d/minivmac/screens/Ubuntu804.gif</image>
+    </screenshot>
+  </screenshots>
+</component>
```
Comment (hfiguiere) [inline com.gryphel.mini_vmac.metainfo.xml:44]: missing required `url`

**Review by hfiguiere — CHANGES_REQUESTED** [review]
desktop, appdata and icon should be upstreamed for a future release.

File: flathub.json (hunk 1 above)
Comment (rs0vere) [reply flathub.json:2]: Because the upstream did not provide the support of aarch64 in Linux

File: com.gryphel.mini_vmac.metainfo.xml (hunk 4 above)
Comment (rs0vere) [reply com.gryphel.mini_vmac.metainfo.xml:46]: I added it now

File: flathub.json (hunk 1 above)
Comment (hfiguiere) [reply flathub.json:2]: do they support flatpak?

File: com.gryphel.mini_vmac.metainfo.xml (hunk 3 above)
Comment (rs0vere) [reply com.gryphel.mini_vmac.metainfo.xml]: I fixed it, thanks

**Comment by rs0vere** [comment]
bot, build com.gryphel.mini_vmac

File: com.gryphel.mini_vmac.yml (hunk 2 above)
Comment (rs0vere) [reply com.gryphel.mini_vmac.yml]: I have used 128x128 icon now

File: flathub.json (hunk 1 above)
Comment (rs0vere) [reply flathub.json:2]: Yes, they all support them, and they are all graphics applications

Comment (hfiguiere) [reply flathub.json:2]: so why not build aarch64 then?

Comment (rs0vere) [reply flathub.json:2]: The build target in the build script provided upstream does not have aarch64, only arm

Comment (rs0vere) [reply flathub.json:2]: This is the supported targets https://www.gryphel.com/c/minivmac/options.html#option_t

**Comment by rs0vere** [comment]
I now think that this emulator does not need to be equipped with something as heavy as Flatpak, to be honest, this is more suitable for users to compile and run by themselves, and I now think that this PR is useless

**Pull request closed without merging**